res/configuration.toml

```diff
@@ -52,3 +51,9 @@
     UpdateLastConnected = false
     [Writable.Device.Discovery]
     Enabled = false
+  [Writable.InsecureSecrets]
```

Comment:
Also need the `SecretStore` cinfig section which should be:

```
[SecretStore]
Type = "vault"
Protocol = "http"
Host = "localhost"
Port = 8200
Path = "device-coap/"
TokenFile = "/tmp/edgex/secrets/device-coap/secrets-token.json"
RootCaCertPath = ""
ServerName = ""
  [SecretStore.Authentication]
  AuthType = "X-Vault-Token"
```
Note this is where the `device-coap` is added to the path.